docs/architecture/input-parameters-for-service-classes.md

```diff
@@ -0,0 +1,23 @@
+# Input parameters for Service Classes
+
+## Overview
+
+Previously, in Kyma Runtime, while provisioning Service Class created from API/Event Definition, there were no possibility to pass input parameters. The ability may be useful in some specific use cases. This document describes passing input parameters from Kyma Runtime to Application or Integration System.
+
+## Solution
+
+The solution utilizes labels on Application level. User provides JSON schema for Service Class `instanceCreateParameterSchema` parameter by creating new Label Definition in the following format:
+
+ **Key**: 
+    - For API Definition: `instance-create-params.apis.{api-name}`
+    - For Event Definition: `instance-create-params.{api-name}.instance-create-params`
```

Comment:
Not sure if the name is valid, shouldn't it follow this format:
`instance-create-params.events.{event-id}`